media_driver/agnostic/gen11/codec/kernel_free/README.md

```diff
@@ -0,0 +1,31 @@
+## DS and HME Kernel compile and convert it to .c
+
+### Step1: Building CM Compiler
+
+1. Download or clone cm compiler
+
+   https://github.com/intel/cm-compiler
+
+2. Build cmc following the steps in readme @https://github.com/intel/cm-compiler
+
+3. Download or clone intel graphics comiler
+
+   https://github.com/intel/intel-graphics-compiler
+
+4. Build igc following the steps in readme @ https://github.com/intel/intel-graphics-compiler
+
+### Step2: Building Kernels
+
+1. Set envrionment variale (Set compiler path)
+
+   export PATH=$PATH:<cm_compiler_path>/include:<cm_compiler_path>/bin
+
+2. run build.py to generate kernel binary
```

Comment:
Do we still use build.py in our patch? We already have merge.py.

Reply:
we don't use build.py in cmake integration, but I did not want to rewrite this README.md: merge.py is not self-contained, we can't just replace build.py with merge.py. So, I decided to left build.py.

We can consider drop this README.md at all...